.github/workflows/replay-verify.yaml

```diff
@@ -23,11 +23,7 @@ on:
       - ".github/workflows/replay-verify.yaml"
       - "testsuite/replay_verify.py"
   schedule:
-    - cron: "0 22 * * *"
-  push:
-    branches:
-      - aptos-release-v* # the aptos release branches
-      - aptos-node-v*
+    - cron: "0 22 * * 1,3,5"
```

Comment:
People talked about running these on 0,2,4 (Sun, Tues, Thurs) so people can debug on Mon, Weds and Fri